src/main/webapp/mapSelection.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Loads available chat rooms to the user
+  */
+function loadChatRooms() { 
```

Comment:
Is it loading only 1 room?

Reply:
For now, we are manually loading a single room, but our intention is that in the coming weeks, we can fully develop functionality to fetch the rooms assigned to the user.